region/lowernorfair/east.json

```diff
@@ -211,7 +214,10 @@
               {
                 "length": 12,
                 "requires": [
-                  {"resetRoomAtNode": [1, 4]},
+                  {"resetRoom":{
+                    "nodes": [1, 2, 3, 4],
```

Comment:
I'm guessing all 4 is here in preparation for an eventual clip

Reply:
Yep, I guess we could remove it if we conclude the clip can't be done but I don't expect that.